src/components/invite/Invite.js

```diff
@@ -20,13 +20,12 @@ import { generateShareObject, isSharingAvailable } from '../../lib/share'
 import AsyncStorage from '../../lib/utils/asyncStorage'
 import ModalLeftBorder from '../common/modal/ModalLeftBorder'
 import { useCollectBounty, useInviteCode, useInvited } from './useInvites'
-import HowToSVG from './howto.svg'
 
 const log = logger.child({ from: 'Invite' })
 
 const shareTitle = 'I signed up to GoodDollar. Join me.'
 const shareMessage =
-  "Hey,\nCheck out GoodDollar it's a digital coin that gives anyone who joins a small daily income (UBI).\n\n"
+  'Hi!\nThis is my referral link to be among the first people to get real, free digital basic income called GoodDollar!\nYou’ll receive a 50 G$ bonus, and join the thousands of people across the globe, building a better, more prosperous future, using GoodDollar.\n\n'
```

Comment:
line is too long. need to split it via `+` concatenation or using ``` literals

Reply:
fixed